components/NavButtons/HowItWorks.tsx

```diff
@@ -1,8 +1,4 @@
-import { Link, Button } from "@chakra-ui/react";
+import Button from "components/shared/Button/Button";
```

Comment:
Minor thing - the purpose of the `/index.ts` file within the `/Button` directory is to allow you to allow us to import the our button from `components/shared/Button`